wicket-extensions/src/main/java/org/apache/wicket/extensions/markup/html/form/datetime/DateLabel.java

```diff
@@ -227,14 +216,10 @@ public DateLabel(String id, DateConverter converter)
 	 * @param converter
 	 *            The converter to use
 	 */
-	public DateLabel(String id, IModel<Date> model, DateConverter converter)
+	public DateLabel(String id, IModel<LocalDate> model, ZonedDateTimeConverter converter)
```

Comment:
This looks wrong: DateLabel works on a LocalDate now, but uses a ZonedDateTimeConverter which converts to/from ZonedDateTime. IMHO we should change this to LocalDate, or we could just delete this class, since it is just a collection of factory methods plus this weird "before" and "after" rendering.

Reply:
+1 for delete